windows_containers/wmco_rn/windows-containers-release-notes-9-x-past.adoc

```diff
@@ -1,37 +1,20 @@
-:_content-type: ASSEMBLY
-[id="windows-containers-release-notes-9-x"]
-= {productwinc} release notes
+:_mod-docs-content-type: ASSEMBLY
+:context: windows-containers-release-notes-9-x-past
```

Comment:
The `:context:` line typically goes after the assembly title. I don't think the placement here will cause any issues (and I didn't notice anything off in the preview), but just noting it just in case.